python/tvm/contrib/target/vitis_ai_utils.py

```diff
@@ -0,0 +1,36 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Vitis AI target utilities"""
```

Comment:
I would sugest we lazily import `pyxir` in target/vitis_ai.py so that it won't be needed even we import vitis_ai.py. As a result, we don't need this file. In fact, we should do this for all partition functions that need 3rd party packages to make sure users won't need to setup every packages even they only need one of them.

Reply:
I see, you prefer doing it like here: https://github.com/apache/tvm/blob/068fed94cf3468e3df510ac8a9aed635ed746804/python/tvm/auto_scheduler/cost_model/xgb_model.py#L114? But then I think we should remove the initialization functions to avoid having unused functionality around. To me this approach is also fine. @leandron , any thoughts?

Reply:
Yes